htsohm/htsohm.py

```diff
@@ -170,8 +171,10 @@ def mutate(run_id, generation, parent):
 
     if mutation_strength:
         print("Mutation strength already calculated for this bin and generation.")
+        sys.stdout.flush()
     else:
         print("Calculating mutation strength...")
+        sys.stdout.flush()
```

Comment:
It's a little crazy to run sys.stdout.flush() after basically EVERY print. It would be better to find an appropriate place to run it once per material, or use the global param, or something.